pages/common/hg.md

```diff
@@ -0,0 +1,19 @@
+# hg
+
+> The `hg` command provides an interface for Mercurial, a distributed source control management system.
```

Comment:
Sorry for nitpicking, but I don't think we need to mention the command's name `hg` in the description, since it's already in the title.

Other than that it looks ok to me!